### ISSUE TITLE
Support reject and doesNotReject method

### ISSUE THREAD

**Issue by falsandtru** [power-assert-js/power-assert#107]
Added since Node 10.

https://nodejs.org/api/assert.html

**Comment by twada** [comment]
@falsandtru Sorry for my late response. I'm now working on this feature.

**Comment by MaximusHaximus** [comment]
Any ETA? :)

**Comment by MaximusHaximus** [comment]
Is this still being worked on?

**Comment by twada** [comment]
@MaximusHaximus Yes I'm working on it. Will be power-assert v2. I'm planning to release it on Nov 23, 2018 at Nodefest Tokyo. So please stay tuned.

**Comment by chuyik** [comment]
Sorry to ping @twada but is there any plan to release v2 anytime soon?
